CRM/Uimods/Tools/SearchTableAdjustments.php

```diff
@@ -379,5 +379,31 @@ public static function getBankAccounts($bank_account_ids) {
     }
     return $baId2reference;
   }
-}
 
+  /**
+   * Modify the events search result table
+   */
+  public static function adjustEventTable($objectName, &$headers, &$rows, &$selector) {
+    foreach ($rows as $key => $row) {
+      // add primary email address to name GP-16672
+      try {
+        $query = \Civi\Api4\Email::get()
+          ->addSelect('email')
+          ->addWhere('contact_id', '=', $row['contact_id'])
+          ->addWhere('is_primary', '=', TRUE)
+          ->setLimit(1);
+        $result = $query
+          ->setCheckPermissions(FALSE)
+          ->execute()
+          ->first();
+        // avoid duplicate (if contact has no name, primary email is set as display name)
+        if (isset($result['email']) && $result['email'] != $row['sort_name']) {
+          $rows[$key]['sort_name'] .= '&nbsp;&lt;' . $result['email'] . '&gt;';
```

Comment:
Let's separate `sort_name` and email with a `<br>` instead (without the brackets), that makes it a bit easier to visually "scan" the results.